### PR TITLE
Add React version to Tiny Quickstart

### PR DESCRIPTION
**Pull request by ghernandez-plaid**

### CODE REVIEW

File: react/README.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+```bash
+cp .env.example .env
+```
+
+Fill out the contents of the **.env** file with the [client ID and Sandbox secret in your Plaid dashboard](https://dashboard.plaid.com/team/keys). Don't place quotes (`"`) around the credentials (i.e., `PLAID_CLIENT_ID=adn08a280hqdaj0ad`). Use the "Sandbox" secret when setting the `PLAID_SECRET` variable.
+
+#### Start the server
+
+Start the app by running the following command:
```
Comment (phoenixy1) [inline react/README.md]:
```suggestion
```
I'm not necessarily going to edit the entire file to conform to this style, but this is an example of how going forward you can be more concise and not spell out things the target audience probably knows.

File: react/README.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+```bash
+yarn start
+```
+
+The app will run on port 3000. To use the app, navigate to `localhost:3000` in your browser. The React frontend and the Node server will hot-reload if you make edits.
```
Comment (phoenixy1) [inline react/README.md]:
```suggestion
The app will run on port 3000 and will hot-reload if you make edits.
```
I tried to make this edit part of the other suggestion but GitHub was confused by the formatting and couldn't handle it.

File: react/README.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+The app allows you to link a sample bank account at an OAuth bank or a non-OAuth bank. For more information on how to link accounts at both types of banks, see the sections below.
+
+#### Non-OAuth banks
+
+Most banks returned by Link in the app are non-OAuth banks. When connecting a non-OAuth bank account, use the following sample credentials:
+
+- Username: `user_good`
+- Password: `pass_good`
+
+If you select a bank that requires multi-factor authentication, enter the following code to proceed: `1234`
```
Comment (phoenixy1) [inline react/README.md]:
```suggestion
If prompted to provide a multi-factor authentication code, use `1234`
```

File: react/README.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+6. Navigate to `localhost:3000` and proceed to link an account.
+
+7. On the "Select your bank" screen, type "oauth" into the search bar. Select "Platypus OAuth Bank".
+
+8. On the next screen, select the first instance of "Platypus OAuth Bank". 
+
+9. Click "Continue" when prompted. You'll be redirected to the login page for "First Platypus Bank". **Important**: Credentials are not necessary for this bank. Simply click "Sign in" to proceed.
+
+10. Continue through the subsequent screens.
```
Comment (phoenixy1) [inline react/README.md:82]: this step doesn't need to be here

File: react/README.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+11. Link will connect the account at the OAuth bank, prompt you to continue, and then redirect you back to the home page.
```
Comment (phoenixy1) [inline react/README.md:84]: this step could be combined with step 9.

**Review by phoenixy1 — COMMENTED** [review]
Nice!

I didn't do a full concision edit, but I made a couple of example concision edits. Following that example, in the spirit of tl;dr, could you do an editing pass to remove any content that is not necessary? Let me know if you have questions, hopefully the examples should make it clear what I am looking for. Thanks!

File: react/README.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+5. Save your changes to the file and restart your local server (i.e., end the current server process and run `yarn start` again).
+
+6. Navigate to `localhost:3000` and proceed to link an account.
+
+7. On the "Select your bank" screen, type "oauth" into the search bar. Select "Platypus OAuth Bank".
+
+8. On the next screen, select the first instance of "Platypus OAuth Bank". 
+
+9. Click "Continue" when prompted. You'll be redirected to the login page for "First Platypus Bank". **Important**: Credentials are not necessary for this bank. Simply click "Sign in" to proceed.
```
Comment (phoenixy1) [inline react/README.md]:
```suggestion
9. Click "Continue" when prompted. You'll be redirected to the login page for "First Platypus Bank". Click "Sign in" to proceed.
```
concision

File: react/yarn.lock [hunk 7]
```diff
@@ -0,0 +1,12401 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (phoenixy1) [inline react/yarn.lock:1]: Having a 12,000 line yarn.lock file in the tiny quickstart is probably not totally consistent with the tiny vibe :-). Is it possible to regenerate yarn.lock to just have what's needed for this project? (I think package.json might have some unneeded deps too, and would be good to audit, but not nearly to this extent.) BTW if the answer is "no" that is fine...I am not super familiar with yarn.

File: react/package.json [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+    "react-dom": "^17.0.1",
+    "react-plaid-link": "^3.3.0",
+    "react-scripts": "4.0.3",
+    "typescript": "^4.1.2",
+    "web-vitals": "^1.0.1"
+  },
+  "scripts": {
+    "start": "run-p start-client start-server",
+    "start-client": "react-scripts start",
+    "start-server": "nodemon server.js",
```
Comment (phoenixy1) [inline react/package.json:28]: Not sure if all these deps are used / you could probably remove some

**Comment by ghernandez-plaid** [comment]
I'm closing this pull request for now due to some awful merge conflicts I'm getting @phoenixy1. Will reopen another shortly.

File: react/yarn.lock (hunk 7 above)
Comment (ghernandez-plaid) [reply react/yarn.lock:1]: Yeah, unfortunately this is just how Yarn works. It looks like the lock file not only include dependencies, but sub-dependencies as well (dependencies that "top-level" dependencies rely on). The Yarn docs also recommend checking the entire file into source control: https://classic.yarnpkg.com/lang/en/docs/yarn-lock/#toc-check-into-source-control. I did remove irrelevant dependencies, though.

**Pull request closed without merging**